### ISSUE TITLE
Install Python files e.g. for plotting

### ISSUE THREAD

**Issue by wxmerkt** [loco-3d/ndcurves#16]
Currently, only the bindings themselves are installed - not the files used in examples e.g. plot_bezier

**Comment by stonneau** [comment]
Hi,
everything in the "sandbox" folder is indeed considered as experimental, so it is true that those methods are not installed, essentially because I was not sure their implementation was the correct one. But it is true that for plotting it would be nice to at lease have a few methods. I ll make the modifications asap. Thanks

Steve

**Comment by wxmerkt** [comment]
Thank you :-)

**Comment by stonneau** [comment]
Hi, this has been implemented. @nim65s is currently checking an internal pull request prior to a binary release on curves. 

However with respect to the jupyter notebook examples I decided only to expose the plotting functions, not the optimization ones that should not be part of the library. The methods are thus reimplemented in the notebook.

Once it is validated this issue will be resolved and the public PR referenced.
Sincerely
